versioned_docs/version-2.5.1/admin-api-namespaces.md

```diff
@@ -612,7 +612,7 @@ $ pulsar-admin namespaces split-bundle --bundle 0x00000000_0xffffffff test-tenan
 
 ```java
 
-admin.namespaces().splitNamespaceBundle(namespace, bundle)
+admin.namespaces().splitNamespaceBundle(namespace, bundle, unloadSplitBundles, splitAlgorithmName)
```

Comment:
The `splitAlgorithmName ` parameter is added in version 2.5.1.

Reply:
The modifications have been completed.